www/includes/easyparliament/templates/html/section/day.php

```diff
@@ -0,0 +1,89 @@
+<div class="full-page__row">
+
+    <div class="business-section">
+        <div class="business-section__header">
+            <h1 class="business-section__header__title">
+                <?= $parent_title ?>
+            </h1>
+            <p class="business-section__header__date">
+                <?= format_date( $info['date'], LONGERDATEFORMAT ) ?>
+            </p>
+            <div class="business-section__header__description">
+                <?php include '_' . $section . '_desc.php'; ?>
+            </div>
+        </div>
+        <?php if ( isset($rows) ) { ?>
+        <div class="business-section__primary">
+            <ul class="business-list">
+            <?php
+                $first = 0;
+                foreach ( $rows as $row ) {
+                    if ( $row['htype'] == 10 ) {
+                        if ( !$first ) {
+                            print '</li>';
+                            $first = 1;
+                        } ?>
+                        <li>
+                        <?php if ( isset($row['excerpt']) && strstr($row['excerpt'], "was asked&#8212;") ) { ?>
+                            <div class="business-list__title">
+                                <h3>
+                                    <?= $row['body'] ?>
+                                </h3>
+                            </div>
+                        <?php } else { ?>
+                            <a href="<?= $row['listurl'] ?>" class="business-list__title">
+                                <h3>
+                                    <?= $row['body'] ?>
+                                </h3>
+                            </a>
+                        <?php }
+                        if ( isset($row['contentcount']) && $row['contentcount'] > 0 ) { ?>
+                            <p class="business-list__meta">
+                                <?= $row['contentcount'] == 1 ? '1 speech' : $row['contentcount'] . ' speeches' ?>
+                            </p>
+                        <?php }
+                        if ( isset($row['excerpt']) ) { ?>
+                            <p class="business-list__excerpt">
+                                <?= trim_characters($row['excerpt'], 0, 200 ) ?>
+                            </p>
+                        <?php }
+                    } else { ?>
+                    <ul>
```

Comment:
This is creating a sub list by creating a new 1-item list for each entry in the sublist, which doesn't seem right.